src/GPSPoint.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Validator;
+
+final class GPSPoint extends AbstractValidator
+{
+
+    const OUT_OF_BOUNDS = 'gpsPointOutOfBounds';
+    const CONVERT_ERROR = 'gpsPointConvertError';
+    const INCOMPLETE_COORDINATE = 'gpsPointIncompleteCoordinate';
+
+    /**
+     * @var array
+     */
+    protected $messageTemplates = [
+        'gpsPointOutOfBounds' => '%value% is out of Bounds.',
+        'gpsPointConvertError' => '%value% can not converted into a Decimal Degree Value.',
+        'gpsPointIncompleteCoordinate' => '%value% did not provided a complete Coordinate',
+    ];
+
+    /**
+     * Returns true if and only if $value meets the validation requirements
+     *
+     * If $value fails validation, then this method returns false, and
+     * getMessages() will return an array of messages that explain why the
+     * validation failed.
+     *
+     * @param  mixed $value
+     * @return bool
+     * @throws Exception\RuntimeException If validation of $value is impossible
+     */
+    public function isValid($value)
+    {
+        if (strpos($value, ',') === false) {
+            $this->error(GPSPoint::INCOMPLETE_COORDINATE, $value);
+            return false;
+        }
+
+        list($lat, $long) = explode(',', $value);
+
+        if ($this->isValidCoordinate($lat, 90.0000) && $this->isValidCoordinate($long, 180.000)) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * @param string $value
+     * @param $maxBoundary
+     * @return bool
+     */
+    private function isValidCoordinate($value, $maxBoundary)
```

Comment:
@freax We've been gradually moving towards private visibility by default. This encourages good encapsulation, and moves the onus for justifying protected/public visibility to consumers; in other words, we need a very, very good reason to make something visible to extension writers.

Doing this eases our maintenance burden, because we can add, change, or remove private methods with little to no impact on end-users.